src/test/scala/example/Book.scala

```diff
@@ -0,0 +1,27 @@
+package example
+
+import org.scalatest._
+import io.circe._, io.circe.parser._, io.circe.generic.auto._, io.circe.syntax._
+
+class BookSpec extends FunSpec {
+  val isbn10: String = "0545790352"
+  val isbn13: String = "0545790352123"
+  val industryIdentifiers = List(
+    IndustryIdentifier("ISBN_10", isbn10),
+    IndustryIdentifier("ISBN_13", isbn13)
+  )
+  val book = Volume(VolumeInfo(Some(industryIdentifiers)))
+
+  describe("fromVolume") {
+    it("should return a new instance of a book given a volume") {
+      assert(Book.fromVolume(book) contains Book(isbn10))
```

Comment:
The naming is a bit confusing. When I read the test it says "fromVolume(..." but the param is a "book". Maybe the "book" in line 13 should be renamed to "volume".